homeassistant/components/hydrawise/binary_sensor.py

```diff
@@ -26,18 +26,18 @@
     device_class=BinarySensorDeviceClass.CONNECTIVITY,
 )
 
-BINARY_SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
-    BinarySensorEntityDescription(
-        key="is_watering",
-        name="Watering",
-        device_class=BinarySensorDeviceClass.MOISTURE,
-    ),
+BINARY_SENSOR_IS_WATERING = BinarySensorEntityDescription(
+    key="is_watering",
+    name="Watering",
```

Comment:
Change to translation key and set translation in `strings.json`

Reply:
Fix this in separate PR also including the one above.
Probably both of them can use device_class translation instead

Reply:
Can I do this in a follow-up PR? `strings.json` is being added in this PR, and based on past experience, getting git to do the right thing on a file-add merge conflict is painful.

Reply:
No, we should do it right away.
For the other platforms that can be done in a follow-up PR

Reply:
Sounds good. I haven't touched them yet, so I think this is resolved. Making a note to myself to update these in a follow-up PR. (Among the many other things that need cleaning up here)

Reply:
Done